fastlane/lib/fastlane/swift_lane_manager.rb

```diff
@@ -54,6 +62,11 @@ def self.cruise_lane(lane, parameters = nil, env = nil)
       duration = ((Time.now - started) / 60.0).round
 
       finish_fastlane(nil, duration, e, skip_message: skip_message)
+
+      if display_upgraded_message
+        UI.message("We updated your FastlaneRunner project during this run to make it compatible with your current version of fastlane.")
+        UI.message("Please make sure to check the changes into source control.")
```

Comment:
What happens after that? That's during a normal _fastlane_ run, right? Will the user be able to see this, or does it pretty much immediately disappear because the actual build output will be printed out?

Reply:
This happens at the end of the run 🎉 

<img width="799" alt="screen shot 2018-01-10 at 10 25 28 am" src="https://user-images.githubusercontent.com/1304821/34788833-a5c890d6-f5f0-11e7-9127-b82d681b658d.png">